[Bug] [jdbc] sql模式配置执行后post-sql 报sql解析错误

### Search before asking

- [X] I had searched in the [issues](https://github.com/DTStack/chunjun/issues) and found no similar issues.


### What happened

sql模式，当写入mysql时，如果配置了写入后需要执行的sql时会报sql解析错误

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.CommandLineWrapper.main(CommandLineWrapper.java:64)
Caused by: com.dtstack.chunjun.throwable.ChunJunRuntimeException: com.dtstack.chunjun.throwable.ChunJunSqlParseException: 
----------sql start---------
1>    
2>    
3>    insert into sink select * from source

----------sql end--------- 

Cannot find table '`default_catalog`.`default_database`.`sink`' in any of the catalogs [default_catalog], nor as a temporary table.
	at com.dtstack.chunjun.Main.exeSqlJob(Main.java:180)
	at com.dtstack.chunjun.Main.main(Main.java:131)
	at com.dtstack.chunjun.local.test.LocalTest.main(LocalTest.java:193)
	... 5 more
Caused by: com.dtstack.chunjun.throwable.ChunJunSqlParseException: 
----------sql start---------
1>    
2>    
3>    insert into sink select * from source

----------sql end--------- 

Cannot find table '`default_catalog`.`default_database`.`sink`' in any of the catalogs [default_catalog], nor as a temporary table.
	at com.dtstack.chunjun.sql.parser.SqlParser.lambda$parseSql$1(SqlParser.java:67)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at com.dtstack.chunjun.sql.parser.SqlParser.parseSql(SqlParser.java:61)
	at com.dtstack.chunjun.Main.exeSqlJob(Main.java:156)
	... 7 more
Caused by: org.apache.flink.table.api.TableException: Cannot find table '`default_catalog`.`default_database`.`sink`' in any of the catalogs [default_catalog], nor as a temporary table.
	at org.apache.flink.table.catalog.CatalogManager.lambda$getTableOrError$0(CatalogManager.java:377)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at org.apache.flink.table.catalog.CatalogManager.getTableOrError(CatalogManager.java:374)
	at org.apache.flink.table.planner.operations.SqlToOperationConverter.convertSqlInsert(SqlToOperationConverter.java:824)
	at org.apache.flink.table.planner.operations.SqlToOperationConverter.convertValidatedSqlNode(SqlToOperationConverter.java:351)
	at org.apache.flink.table.planner.operations.SqlToOperationConverter.convert(SqlToOperationConverter.java:262)
	at org.apache.flink.table.planner.delegation.ParserImpl.parse(ParserImpl.java:106)
	at org.apache.flink.table.api.internal.StatementSetImpl.addInsertSql(StatementSetImpl.java:62)
	at org.apache.flink.table.api.bridge.java.internal.StreamStatementSetImpl.addInsertSql(StreamStatementSetImpl.java:45)
	at org.apache.flink.table.api.bridge.java.internal.StreamStatementSetImpl.addInsertSql(StreamStatementSetImpl.java:30)
	at com.dtstack.chunjun.sql.parser.InsertStmtParser.execStmt(InsertStmtParser.java:42)
	at com.dtstack.chunjun.sql.parser.AbstractStmtParser.handleStmt(AbstractStmtParser.java:43)
	at com.dtstack.chunjun.sql.parser.AbstractStmtParser.handleStmt(AbstractStmtParser.java:45)
	at com.dtstack.chunjun.sql.parser.AbstractStmtParser.handleStmt(AbstractStmtParser.java:45)
	at com.dtstack.chunjun.sql.parser.SqlParser.lambda$parseSql$1(SqlParser.java:64)
	... 18 more

```

### What you expected to happen

希望可以正常执行写入后的sql

### How to reproduce


```

CREATE TABLE source
(
    username varchar,
    age          int
) WITH (
      'connector' = 'stream-x',
      'number-of-rows' = '1'
);

CREATE TABLE sink
(
    username varchar,
    age          int
) WITH (
      'connector' = 'mysql-x',
      'url' = 'jdbc:mysql://ip:3306/flinkx_test?useUnicode=true&characterEncoding=utf8&useSSL=false',
      'table-name' = 'flinkx_dm8_test',
      'username' = 'user',
      'password' = 'pwd',
      'sink.buffer-flush.max-rows' = '1024',
      'sink.buffer-flush.interval' = '10000',
      'sink.all-replace' = 'false',
      'sink.parallelism' = '1'
      ,'sink.pre-sql' = 'delete from flinkx_dm8_test where id=3'
      ,'sink.post-sql' = 'update flinkx_dm8_test set username=''李四-2'' where id=4'
      );

insert into sink select * from source;


```

### Anything else

_No response_

### Version

master

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/chunjun-core/src/main/java/com/dtstack/chunjun/sql/parser/SetStmtParser.java
+++ b/chunjun-core/src/main/java/com/dtstack/chunjun/sql/parser/SetStmtParser.java
@@ -22,6 +22,8 @@ import org.apache.flink.configuration.Configuration;
 import org.apache.flink.table.api.StatementSet;
 import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
 
+import org.apache.commons.lang3.StringUtils;
+
 import java.net.URL;
 import java.util.List;
 import java.util.regex.Matcher;
@@ -39,7 +41,9 @@ public class SetStmtParser extends AbstractStmtParser {
 
     @Override
     public boolean canHandle(String stmt) {
-        return SET_PATTERN.matcher(stmt).find();
+        return StringUtils.isNotBlank(stmt)
+                && stmt.trim().toLowerCase().startsWith("set")
+                && SET_PATTERN.matcher(stmt).find();
     }
 
     @Override

